src/angularComponent/ngcModifySuggestionsTemplateFactory.js

```diff
@@ -69,6 +69,9 @@ export default function ngcModifySuggestionsTemplateFactory($document, $template
       itemEl.setAttribute('ng-attr-aria-selected', '{{suggestionItem.isHighlighted()}}');
       itemEl.setAttribute('ng-attr-aria-readonly',
           '{{!suggestionItem.isSelectable() || undefined}}');
+      itemEl.setAttribute('ng-mouseenter', 'omnibox.highlightItem(suggestion)');
+      itemEl.setAttribute('ng-mouseleave', 'omnibox.highlightNone()');
```

Comment:
Just realized why not, if a header is disabled, I would expect it to highlight none when leaving the item and hovering the header. If I put this on the parent, it'll stay highlighted since the header and the item share a parent.